play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/cache/CachingAsyncHttpClient.scala

```diff
@@ -290,5 +291,13 @@ class CachingAsyncHttpClient(
   override def executeRequest(requestBuilder: RequestBuilder): ListenableFuture[AHCResponse] = {
     underlying.executeRequest(requestBuilder)
   }
+
+  override def prepare(method: String, url: String): BoundRequestBuilder = underlying.prepare(method, url)
```

Comment:
Are those part of the Java API too now?

Reply:
these are the changes introduced with async-http-client 2.1